src/client.js

```diff
@@ -333,7 +333,7 @@ class Client extends Base {
           result = obj.Body[outputName];
```

Comment:
we can fix the previous regex block as well :
(?:(Out|\.Out)(?:put)?|(Response|\.Response))$